build.sbt

```diff
@@ -13,28 +13,37 @@ homepage := Some(url("https://github.com/codingwell/scala-guice"))
 
 libraryDependencies ++= Seq(
   "com.google.inject" % "guice" % "5.1.0",
-  "org.scala-lang" % "scala-reflect" % scalaVersion.value
+  "org.scalatest" %% "scalatest" % "3.2.9" % "test",
+  "com.google.code.findbugs" % "jsr305" % "3.0.2" % "compile"
 )
 
-libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.9" % "test"
-
-libraryDependencies += "com.google.code.findbugs" % "jsr305" % "3.0.2" % "compile"
-
-libraryDependencies += "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.2"
+libraryDependencies ++= {
+  CrossVersion.partialVersion(scalaVersion.value) match {
+    case Some((2, n)) => Seq(
+      "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.2",
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value
+    )
+    case _ => Seq.empty
+  }
+}
 
 autoAPIMappings := true
 
-scalaVersion := "2.13.8"
+//scalaVersion := "2.13.8"

```

Comment:
Why leave this commented? I think you can just remove it

Reply:
Removed.